packages/checkbox/src/Checkbox.test.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { mount, render } from "enzyme";
+import Checkbox from "./Checkbox";
+
+describe("Checkbox", () => {
+  const subject = (props = {}) =>
+    mount(<Checkbox title="Checkbox" {...props} />);
+  let wrapper;
+  let checkbox;
+
+  describe("integration", () => {
+    beforeEach(() => {
+      wrapper = render(<Checkbox checked={true} />);
+      checkbox = wrapper.find(".hig__input-button");
+    });
+
+    it("renders a presenter", () => {
+      expect(checkbox.length).not.toBe(0);
+    });
+
+    it("input element has checked class when props.checked is true", () => {
+      expect(checkbox.hasClass("hig__input-button--checked"));
+    });
+  });
+
+  describe("event handlers", () => {
+    let eventHandler;
+
+    describe("onBlur", () => {
+      beforeEach(() => {
+        eventHandler = jest.fn();
```

Comment:
_Nitpick:_ Instead of creating a new mock function for every test, consider using `.mockReset()`,  `.mockClear()`, or `jest.resetAllMocks().`

Reply:
Addressed in SHA b861e15d